packages/react-storage/src/components/StorageBrowser/actions/configs/defaults.tsx

```diff
@@ -31,7 +31,7 @@ export const copyActionConfig: CopyActionConfig = {
 export const deleteActionConfig: DeleteActionConfig = {
   componentName: 'DeleteView',
   actionsListItemConfig: {
-    disable: (selected) => !selected,
+    disable: (selected) => !selected || selected.length === 0,
```

Comment:
can selected be `undefined` here?

Reply:
With the initial state, the `selected` is passed as `undefined`.

Reply:
It'd be short circuited by the first part, I think?